nixos/doc/manual/installation/installing.xml

```diff
@@ -46,6 +46,12 @@
    to increase the font size.
   </para>
 
+  <para>
+    To install over a <link xlink:href="https://en.wikipedia.org/wiki/Serial_port">serial port</link>
```

Comment:
Do we need the wikipedia link?

Reply:
I added that link to allow readers to get more context on what a serial port, in case they were unaware whether this documentation applied to them.

I'm happy to remove this link, however.

Reply:
I have removed this link.